services/frontend/src/components/profileForms/primaryInfoForm/PrimaryInfoFormView.jsx

```diff
@@ -488,7 +488,23 @@ const PrimaryInfoFormView = ({
             <Col className="gutter-row" xs={24} md={8} lg={8} xl={8}>
               <Form.Item
                 name="pri"
-                label={<FormattedMessage id="pri" />}
+                label={
+                  <>
+                    <FormattedMessage id="pri" />
+                    <div className="prim-popoverStyleCareer">
+                      <Popover
+                        trigger={["focus", "hover"]}
+                        content={
+                          <div className="prim-popoverStyle">
+                            <FormattedMessage id="pri.private" />
+                          </div>
+                        }
+                      >
+                        <InfoCircleOutlined tabIndex={0} />
```

Comment:
antd has a tool tip built in for form item
![image](https://user-images.githubusercontent.com/11470442/120814284-bae10f80-c51c-11eb-8216-59b9f6933a40.png)